rest/src/main/groovy/whelk/rest/api/ESQuery.groovy

```diff
@@ -466,4 +466,42 @@ class ESQuery {
         }
         return result
     }
+
+    @CompileStatic(TypeCheckingMode.SKIP)
```

Comment:
Nitpick: Javadoc should appear before annotations.

Reply:
D:

I fix!